openlibrary/core/cache.py

```diff
@@ -67,8 +67,12 @@ def _get_memcache(self):
                 self._memcache = memcache.Client(servers)
             else:
                 web.debug("Could not find memcache_servers in the configuration. Used dummy memcache.")
-                import mockcache
-                self._memcache = mockcache.Client()
+                try:
+                    import mockcache  # Only supports legacy Python
+                    self._memcache = mockcache.Client()
+                except ImportError:  # Python 3
+                    from pymemcache.test.utils import MockMemcacheClient
+                    self._memcache = MockMemcacheClient()
```

Comment:
Why are we using a mock memcache client instead of a "real" client?

Reply:
Line 69: web.debug("Could not find memcache_servers in the configuration. Used dummy memcache.")

Reply:
🤦‍♂ Thank you

Reply:
I hope that in prod, it is [ol-mem0, ol-mem1, ol-mem2](https://github.com/internetarchive/openlibrary/wiki/Production-Service-Architecture#current-production-architecture) that are configured.

Reply:
Yep; they're definitely available on prod, and likely also on local-dev actually. (Which begs the question why this code exists at all; presumably just for testing?)